rust/yarp-sys/tests/utils_tests.rs

```diff
@@ -0,0 +1,131 @@
+use std::{
+    ffi::{CStr, CString},
+    mem::MaybeUninit,
+};
+
+#[test]
+fn version_test() {
+    use yarp_sys::yp_version;
+
+    let cstring = unsafe {
+        let version = yp_version();
+        CStr::from_ptr(version)
+    };
+
+    assert_eq!(&cstring.to_string_lossy(), "0.4.0");
```

Comment:
(same question as in another file--shouldn't I be getting 0.5.0? I see `EXPECTED_YARP_VERSION` in `extension.h` is set to "0.4.0", but I thought the gem was at 0.5.0, no? Are these intended to track each other?)